chainstate/src/config.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://spdx.org/licenses/MIT
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use serde::{Deserialize, Serialize};
+
+/// The chainstate subsystem configuration.
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ChainstateConfig {
+    /// The number of maximum attempts to process a block.
+    pub max_db_commit_attempts: usize,
+    /// The maximum capacity of the orphan blocks pool.
+    pub max_orphan_blocks: usize,
```

Comment:
(Not for this PR.) Maybe there should be a limit on the total size of orphan blocks in bytes in addition to this.

Reply:
The total size should come as a consequence of the total number since the size of a single block is capped already.